lib/std/c/haiku.zig

```diff
@@ -282,15 +292,29 @@ pub const dirent = extern struct {
     }
 };
 
+pub const area_info = extern struct {
+    area: u32,
+    name: [32]u8,
```

Comment:
[32] is from the B_OS_NAME_LENGTH constant. I don't know if that should be preserved here or not...

Reply:
i would be open to suggestions rather sticking with this hard-coded value (perhaps use a constant with corresponding name?). another suggestion was to include many of the B_* constants that after taking a brief look at it, it might be better split off into another more directed changeset.